apps/emqx_retainer/src/emqx_retainer_schema.erl

```diff
@@ -47,3 +56,23 @@ is_pos_integer(V) ->
 
 backend_config() ->
     #{type => hoconsc:union([hoconsc:ref(?MODULE, mnesia_config)])}.
+
+retainer_indexes(type) -> list(list(integer()));
+retainer_indexes(desc) ->
+    "Retainer index specification: list of arrays of positive ascending integers";
```

Comment:
From user perspective this description is quite cryptic, since it doesn't explain how this parameter affects the system and how to configure it.

Reply:
Yes, we should update this if decide keep index specifications in config.